src/main/text files/MidtermSurvey

```diff
@@ -0,0 +1,23 @@
+Mid-Term Survey
+
+- What are you struggling the most in this class? Name some topics that you had the most difficulties in understanding.
+    - I had a really hard time understanding how to implement linked lists and stacks. I understood the concepts but
+    I just couldn't figure out how to set them up or how to use them in code. For example with stacks I had a hard
+    time figuring out what I could do with stacks, or how to implement them since I didn't know the format of how they
+    worked. Also I think it's just really hard to get everything done since the lab that we take as co-requisite for this
+    course gives us labs that are also just as difficult.
```

Comment:
we have helps from me and others on discord. please reach out for help. it's even okay to just say "I don't understand this and that at all. can we discuss it?"